### ISSUE TITLE
Bug in recps property

### ISSUE THREAD

**Issue by christianbundy** [ssbc/ssb-schema-definitions#2]
Note to myself to fix this for Oasis!

Found this lovely `recps` in the wild:

```js
{
  // [...]
  "recps": [
    "@abc[...]",
    {
      "link": "@def[...]",
      "name": "dan"
    },
    {
      "link": "@ghi[...]",
      "name": "gabby"
    },
    {
      "link": "@jkl[...]",
      "name": "jerry"
    },
    {
      "link": "@mno[...]",
      "name": "marsha"
    }
  ]
}
```

It looks like ssb-schema-definitions is saying that this is not a valid `recps`, probably because of the lone `@abc`.

**Comment by christianbundy** [comment]
Solution seems to be un-commenting this line: https://github.com/ssbc/ssb-schema-definitions/blob/master/definition/primitives.js#L49

**Comment by dominictarr** [comment]
yeah, you should accept a mix of both of these but preferably only accept an array of feed ids... (just checking, looks like that is the typical format)

**Comment by christianbundy** [comment]
More discussion here, btw: https://github.com/ssbc/ssb-schema-definitions/issues/2
